resources/user-guides/signal-extension-and-rules-engine-integration.md

```diff
@@ -63,15 +59,15 @@ ACPCore.collectPii(piiContextData)
 {% endtab %}
 {% endtabs %}
 
-1. In Adobe Experience Platform Launch, create a Data element for the `user_email` context data key.
+1. In Adobe Experience Platform Launch, create a data element for the `user_email` context data key.
 
-![Data Element Example for Collect PII context data key](../../../.gitbook/assets/data_element_example_collect_pii.png)
+![Data Element Example for Collect PII context data key](img/data_element_example_collect_pii.png)
 
 1. In Experience Platform Launch, create a new rule for sending a postback.
 
 Create a new rule by selecting the Event Mobile Core Collect PII and the Action Mobile Core Send Postback as in the following image:
```

Comment:
Please reword as follows:
"Create a new rule by selecting the **Mobile Core Collect PII** event and the **Action Mobile Core Send Postback** action as illustrated in the following graphic: